universal-login-relayer/lib/config/knexfile.ts

```diff
@@ -24,7 +24,7 @@ function getTestKnexConfig(): KnexConfig {
     client: 'postgresql',
     connection: {
       database: 'universal_login_relayer_test',
-      user:     'postgres',
+      user: 'postgres',
```

Comment:
The purpose of this PR should be to get rid of `get*KnexConfig` at all. All data should be inlined.